data/mods/Aftershock/items/ammo/plasma.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "id": "afs_shydrogen",
+    "type": "AMMO",
+    "name": { "str_sp": "solid hydrogen" },
+    "description": "Pressurized and cryo-cooled rods of solidified hydrogen, used within plasma weapons or portable powerplants.",
+    "weight": "1 g",
+    "volume": "12 ml",
+    "price": "10 USD",
+    "price_postapoc": "10 USD",
+    "material": [ "hydrogen_cell" ],
+    "symbol": "=",
+    "color": "light_blue",
+    "phase": "gas",
```

Comment:
Gas?

Reply:
correct; plasma is usually a gas

Reply:
Yeah its basically to ensure the player cant get solid hydrogen out of its canisters. No cooling your drinks with hydrogen ice-cubes.